cli/module_registry.go

```diff
@@ -81,17 +76,20 @@ func CreateModuleCommand(c *cli.Context) error {
 		return errors.Wrap(err, "failed to register the module on app.viam.com")
 	}
 
-	fmt.Fprintf(c.App.Writer, "Successfully created '%s'.\n", response.GetModuleId())
-	if response.GetUrl() != "" {
-		fmt.Fprintf(c.App.Writer, "You can view it here: %s \n", response.GetUrl())
-	}
-
 	returnedModuleID, err := parseModuleID(response.GetModuleId())
 	if err != nil {
 		return err
 	}
-	if returnedModuleID.Prefix == "" && org.PublicNamespace != "" {
-		returnedModuleID.Prefix = org.PublicNamespace
+	// The registry team is currently of the opinion that including an org id in the meta.json file
+	// is non-ideal.
+	// If you do change this, edit the UpdateCommand().. function to also check if the manifestprefix is an orgid
```

Comment:
```suggestion
	// If you do change this, edit the UpdateCommand().. function to also check if the manifest prefix is an orgid
```
overall fine with the comment :)